src/index.js

```diff
@@ -30,35 +32,53 @@ const App = () => {
 
     useEffect(() => {
         localStorage.setItem(LOCAL_STORAGE_KEY_DATA, JSON.stringify(todos));
-    }, [todos]); // this effect only fires if the todos array changes: which is toggle complete and remove tasks
+        enableClearButton();
```

Comment:
There is a need to enable the `Clear Completed Items `button when the app first loads up, if an item has been checked completed but the user doesn't want it to be removed from the list yet. And the reason why I call `enableClearButton()` here instead of calling it inside the useEffect on line 27, which is when the app first loads up, is because this useEffect calls setTodos anyways, so why not use it in this useEffect which is whenever [todos] changes.